plugins/orchestrator-backend/src/service/SonataFlowService.test.ts

```diff
@@ -155,32 +148,23 @@ describe('SonataFlowService', () => {
     beforeEach(() => {
       jest.clearAllMocks();
     });
-    it('should return workflow execution response when the request is successful', async () => {
+    it('should propagate thrown error when the request is successful', async () => {
       // Given
       setupTest({ ok: true, json: { id: definitionId, status: 'completed' } });
 
       // When
-      const result = await sonataFlowService.executeWorkflow({
-        definitionId,
-        serviceUrl,
-        inputData: { var1: 'value1' },
-      });
-
-      // Then
-      expect(fetch).toHaveBeenCalledWith(
-        urlToFetch,
-        expectedFetchRequestInit(),
-      );
-      expect(result).toEqual({ id: definitionId, status: 'completed' });
```

Comment:
why removing this?

Reply:
it is an error flow. we don't need to check for completed it is not the target of the test. expecting the fetch is absurd. we tell the mock to expect it and it will validate it . so why double test?